presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -145,6 +147,8 @@
     private boolean replay;
     private Duration recordingDuration = new Duration(0, MINUTES);
 
+    private List<String> readAccessByNameSerdeList = ImmutableList.of();
```

Comment:
Any way this can be removed? Adding another property means different installations will have different behaviors and bugs may affect only some people.

If necessary, choose a reasonable default so that set is easier for new users.

Reply:
I can add some reasonable defaults, specifically the default JSON reader that is packaged with Presto. This setting is meant for those that add additional Hive Serde's that can be installed, like Avro, that is not packaged with Presto.